plugins/static-pages.lisp

```diff
@@ -13,13 +13,16 @@
 (in-package :coleslaw-static-pages)
 
 (defclass page (content)
-  ((title :initarg :title :reader coleslaw::title-of)))
+  ((title :initarg :title :reader coleslaw::title-of)
+   (format :initarg :format :reader coleslaw::page-format))
+  ;; default format is markdown (for backward compatibility)
+  (:default-initargs :format "md"))
```

Comment:
It would make more sense to use a keyword here (`:md`). It is what is expected by `render-text`. The initialize-instancer after method is to insure a keyword. `string-upcase`takes a string designator so passing a keyword is ok.

 `(string-upcase :md) `= > "MD"

Other than this, this looks good to merge